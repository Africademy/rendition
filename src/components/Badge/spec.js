/* globals expect, test */
import React from 'react'
import renderer from 'react-test-renderer'
import { Badge, Provider } from '../../../dist'

test('Badge renders correctly', () => {
  const component = renderer.create(
    <Provider>
      <Badge>Badge</Badge>
    </Provider>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
