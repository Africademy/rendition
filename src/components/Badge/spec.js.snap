// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge renders correctly 1`] = `
.c1 {
  font-family: 'Nunito',Arial,sans-serif;
  font-size: 14px;
  line-height: 1.5;
  box-sizing: border-box;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}

.c3 {
  padding: 2px 5px;
  font-size: 16px;
  color: #fff;
  background-color: #8dc587;
}

.c2 {
  border-radius: 3px;
  display: inline-block;
  min-width: 40px;
  text-align: center;
  font-weight: 800;
  font-style: normal;
  font-stretch: normal;
  -webkit-letter-spacing: 0.5px;
  -moz-letter-spacing: 0.5px;
  -ms-letter-spacing: 0.5px;
  letter-spacing: 0.5px;
}

.c0 {
  font-family: 'Nunito',Arial,sans-serif;
  font-size: 14px;
}

.c0 h1,
.c0 h2,
.c0 h3,
.c0 h4,
.c0 h5,
.c0 h6,
.c0 button {
  font-family: CircularStd,Arial,sans-serif;
}

<div
  className="c0 c1"
>
  <div
    className="c2 c3 "
  >
    Badge
  </div>
</div>
`;
